backend-rust/src/migrations/m0007-canonical-address-and-transaction-search-index.sql

```diff
@@ -0,0 +1,9 @@
+-- Initial setup of canonical address with placeholder data
+ALTER TABLE accounts
+ADD COLUMN canonical_address BYTEA DEFAULT E'\\x00000000000000000000000000000000000000';
```

Comment:
I don't think it makes sense for the column to have a default, it would prevent errors if forgotten in an insert query. I would prefer for it to be nullable during the migration and then NOT NULL at the end.

Reply:
So you would alter it in the migration? Sure.

Reply:
Since canonical addresses have a fixed length (29 bytes), that would be good to enforce:

ADD CONSTRAINT check_canonical_address_length CHECK (LENGTH(canonical_address) = 29);

Reply:
Also I should use UNIQUE and remove the index.